x-pack/plugins/apm/server/routes/services/route.ts

```diff
@@ -129,18 +130,29 @@ const servicesRoute = createApmServerRoute({
         : Promise.resolve(null),
       getRandomSampler({ security, request, probability }),
     ]);
-    const searchAggregatedTransactions = await getSearchAggregatedTransactions({
-      ...setup,
-      kuery,
-      start,
-      end,
-    });
+
+    const [searchAggregatedTransactions, searchAggregatedServiceMetrics] =
+      await Promise.all([
+        getSearchAggregatedTransactions({
+          ...setup,
+          kuery,
+          start,
+          end,
+        }),
+        getSearchAggregatedServiceMetrics({
+          ...setup,
+          kuery,
+          start,
+          end,
+        }),
+      ]);
```

Comment:
Is it something we'll have to reuse in a lot of places? I see it's already doing the same thing on line 228, shouldn't we create a helper function?